bmm/src/main/java/org/openehr/bmm/v2/persistence/PBmmContainerProperty.java

```diff
@@ -14,4 +23,22 @@ public void setCardinality(Interval<Integer> cardinality) {
         this.cardinality = cardinality;
     }
 
+    @Override
+    public BmmContainerProperty createBmmProperty(BmmClassProcessor classProcessor, BmmClass bmmClass) {
+        PBmmContainerType typeRef = getTypeRef();
+        if (typeRef != null) {
+            BmmContainerType bmmType = typeRef.createBmmType(classProcessor, bmmClass);
+            if (bmmType != null) {
+                BmmContainerProperty bmmProperty = new BmmContainerProperty(getName(), bmmType, getDocumentation(), nullToFalse(isMandatory()), nullToFalse(isComputed()));
+                if (getCardinality() != null) {
+                    bmmProperty.setCardinality(new MultiplicityInterval(getCardinality()));
+                }
+                populateImBooleans(bmmProperty);
+                return bmmProperty;
+            }
+        }
+        throw new RuntimeException("BmmTypeCreate failed for type " + typeRef.asTypeString() + " of property "
+                + getName() + " in class " + bmmClass.getName());
```

Comment:
`typeRef` might be `null` at this point, causing `typeRef.asTypeString()` to throw a NullPointerException.